portfolio/src/main/webapp/talent.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <title>Pictures Of Jaden</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <div id="content">
+      <h1 class="center"> Jaden's Fun Fact</h1>
```

Comment:
Remove errant space after the opening `<h1>` tag unless there's a need for it

Reply:
Got it.